contracts/src/libraries/FROST.sol

```diff
@@ -155,30 +258,75 @@ library FROST {
         }
     }
 
+    // ============================================================
+    // PRIVATE FUNCTIONS - HASH DOMAINS
+    // ============================================================
+
+    /**
+     * @notice Hashes input to a field element for binding factors.
+     * @param input The input bytes.
+     * @return result The field element.
+     */
     function _h1(bytes memory input) private view returns (uint256 result) {
         return _hashToField(input, "FROST-secp256k1-SHA256-v1rho\x00\x00\x00\x1c");
```

Comment:
Is it worth mentioning that this string is specific to our implementation/ cyphersuit?